superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -853,8 +853,9 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
     const { hostname } = window.location;
     let ipAlert = connectionAlert?.REGIONAL_IPS?.default || '';
     const regionalIPs = connectionAlert?.REGIONAL_IPS || {};
-    Object.entries(regionalIPs).forEach(([regex, ipRange]) => {
-      if (regex.match(hostname)) {
+    Object.entries(regionalIPs).forEach(([ipRegion, ipRange]) => {
+      const regex = new RegExp(ipRegion);
```

Comment:
you used to have the case insensitive flag on this? Do you want to bring it back in?